### PR TITLE
Add the same functionally to the keypad enter key as the return key (enter key)

### PR DESCRIPTION
**Pull request by bobcanthelpyou**

### CODE REVIEW

File: CorsixTH/Lua/ui.lua [hunk 1]
```diff
@@ -803,7 +804,7 @@ function UI:afterLoad(old, new)
 
   if old < 100 then
     self:removeKeyHandler({"alt", "enter"}, self)
-    self:addKeyHandler({"alt", "Return"}, self, self.toggleFullscreen)
+    self:addKeyHandler({"alt", "return"}, self, self.toggleFullscreen)
```
Comment (TheCycoONE) [inline CorsixTH/Lua/ui.lua]: I'm not sure why you changed this line, or the other place you substituted Return for return. It's not documented in your commit.

**Comment by TheCycoONE** [comment]
Keyboard bindings also need to be added for old save games, at the very least the one for ui.lua.  Bump the savegame number, and add an afterLoad condition to add the new one.

File: CorsixTH/Lua/ui.lua (hunk 1 above)
Comment (bobcanthelpyou) [inline CorsixTH/Lua/ui.lua]: It is just 'consistency', cause every other addKeyHandler is using lowercase key-names, i know addKeyHandler will lowercase all key-names, too. So this is not necessary, but i did it. I hope it does not make problems with the afterLoad stuff for old savegames

**Comment by bobcanthelpyou** [comment]
I just have to add something like:
```
   if old-savegame-version < new-savegame-version then
     self:addKeyHandler("keypad enter", self, ...)
```

**Comment by TheCycoONE** [comment]
yep, and bump the new savegame version (in app.lua)

**Comment by bobcanthelpyou** [comment]
added the new keyhandler to the afterload functions and bump the savegame version: #833 

Did i missed something? Tested most of them with some old savegames, should work hopefully.

**Pull request merged**